src/Our.Umbraco.Ditto/Extensions/TypeInferenceExtensions.cs

```diff
@@ -8,7 +10,7 @@
     /// Extensions methods for <see cref="T:System.Type"/> for inferring type properties.
     /// Most of this code was adapted from the Entity Framework
     /// </summary>
-    internal static class TypeInferenceExtensions
+    public static class TypeInferenceExtensions
```

Comment:
@leekelleher nUnit trade off, always have an inner battle between testing only public methods versus wanting to ensure things like this work. InternalsVisibleTo makes me weep a little inside.